tldw_Server_API/app/api/v1/endpoints/slides.py

```
@@ -603,6 +607,39 @@ def _resolve_provider(request_provider: str | None) -> str:
+    transcript = get_latest_transcription(media_db, media_id)
+    if isinstance(transcript, str) and transcript.strip():
+        return transcript.strip()
+
+    try:
+        latest_document = get_document_version(
+            db_instance=media_db,
+            media_id=media_id,
+            version_number=None,
+            include_content=True,
+        )
+    except Exception as exc:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)